src/index.js

```diff
@@ -70,15 +89,16 @@ export default function deepForceUpdate(
 
   let node = root
   while (true) {
-    if (node.tag === ReactClassComponent && shouldUpdate(node)) {
+    const publicInfo = extractPublicInfoFiber(node);
```

Comment:
Seems unfortunate to call it if it's not used. I guess this is the point where I ask you to change it back to check the function existence inline (and lazily calculate this if either function exists). Sorry!